src/platform/i386/vga.c

```diff
@@ -0,0 +1,204 @@
+#define ENABLE_VGA
+/*
+ * Phani: 09/17/2015
+ *
+ * Code reuse:
+ * from: http://www.osdever.net/bkerndev/index.php 
+ * from: https://www.gnu.org/software/grub/manual/multiboot/multiboot.html#kernel_002ec
+ *
+ */
+
+#include <chal.h>
+#include "kernel.h"
+#include "string.h"
+#include "io.h"
+#include "isr.h"
+
+#define STRLEN_MAX (81)
+
+/* The number of columns. */
+#define COLUMNS                 80
+/* The number of lines. */
+#define LINES                   25
+/* The attribute of an character. */
+#define ATTRIBUTE               0x0F
+/* The video memory address. */
+#define VIDEO                   0xB8000
+
+#define SPACE			0x20
```

Comment:
Line all of these up....I'm not oging to comment on this file as you know it needs to be changed to accommodate cos style.